### ISSUE TITLE
ext.todo todolist not linking to the page in pdflatex

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1020]
I activated the todo extension and use 

.. todolist::

on the first (or last) page of my document.

I created the document using "make latexpdf"

It shows the todo entries well, but the links are always to the first page of the document.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1020
- Originally reported by: Anonymous
- Originally created at: 2012-10-19T17:33:36.546

**Comment by shimizukawa** [comment]
_From Alexander Menk on 2012-10-19 15:35:03+00:00_

issue created by me

**Comment by shimizukawa** [comment]
_From Dietmar Winkler on 2014-08-07 14:27:32+00:00_

The current status is actually that the todolist contains pagreferences to lables that never get set in the created LaTeX document. For example the todo item:

```
\begin{notice}{note}{Todo}

Fix this!

\end{notice}
```

is listed in the todolist as:

```
\begin{notice}{note}{Todo}

Fix this!

\end{notice}

(The {\hyperref[foo/bar/baz_def:index-1]{\emph{original entry}}} 
(\autopageref*{foo/bar/baz_def:index-1}) is located in 
 /path/to/source/of/baz\_def.rst, line 167.)
```

So what is missing is that the orginal todo item should also get a 

``` \label{foo/bar/baz_def:index-1}```

```

**Comment by tk0miya** [comment]
Fixed by #4153!
